janus_server/src/task.rs

```diff
@@ -344,6 +344,9 @@ impl Task {
     }
 }
 
+#[cfg(feature = "test-util")]
+impl Task {}
```

Comment:
Does something break if we don't have an empty `impl`?

Reply:
Oops, nope, this shouldn't be here -- removed. (way too much context: I was initially going to put the new test-only `impl Test` block here, but I realized I could place it directly in the `test_utils` mod instead)